poutyne/framework/experiment.py

```diff
@@ -253,11 +396,53 @@ def train(self,
               batches_per_step=1,
               seed=42):
         # pylint: disable=too-many-locals
-        if seed is not None:
-            # Make training deterministic.
-            random.seed(seed)
-            np.random.seed(seed)
-            torch.manual_seed(seed)
+        """
+        Trains or finetunes the attribute model on a dataset using a generator. If a previous training already occured
+        and lasted a total of `n_previous` epochs, then the model's weights will be set to the best previous checkpoint
+        and the training will be resumed for epochs range (`n_previous`, `epochs`].
+
+        If the Experiment has logging enabled (i.e. self.logging is True) the epoch number, time, loss and metrics on
+        the train and validation sets will all be logged in an entry on an output .tsv file. For a detailed list of all
+        automatically logged metrics, see :func:`here <callbacks.Callback.on_epoch_end()>`.
```

Comment:
I don't think this sentence says anything. We should mention the callbacks we use and why if necessary. For instance, the tensorboard logger, the ModelCheckpoint for the saving the weights of the best epoch, etc.

Reply:
Not sure what you want then. 

Do you want something like "The logging will be done according to [list of Experiment Callbacks linked to their respective doc]." ?

Reply:
I updated the comments, please confirm is that is better.